modules/mri_violations/test/mri_violationsTest.php

```diff
@@ -47,6 +47,13 @@ public function setUp()
              'MRI_alias' => 'test',
             )
         );
+        $this->DB->insert(
+            "subproject",
+            array(
+             'SubprojectID' => '55',
```

Comment:
Not worth blocking the PR for but maybe this could be a constant at the top of the file instead of a magic number?